tests/rados/test_node_drain_customer_bug.py

```diff
@@ -42,19 +42,20 @@ def run(ceph_cluster, **kw):
     service_obj = ServiceabilityMethods(cluster=ceph_cluster, **config)
     ceph_nodes = kw.get("ceph_nodes")
     config = kw["config"]
-
+    # cmd_unset_unmanaged = ""
     replicated_config = config.get("replicated_pool")
     pool_name = replicated_config["pool_name"]
-    active_osd_list = rados_obj.get_osd_list(status="up")
+    active_osd_list = rados_obj.get_active_osd_list()
```

Comment:
Please revert the change in this line to "active_osd_list = rados_obj.get_osd_list(status="up")"
The get_active_osd_list() method no longer exists

Reply:
get_active_osd_list method is the local method  combines the in and up osd list